inference-engine/src/mkldnn_plugin/nodes/mkldnn_topk_node.cpp

```diff
@@ -864,20 +856,20 @@ struct jit_uni_topk_kernel_f32 : public jit_uni_topk_kernel, public jit_generato
             vextractf32x4(xmm_idx(4), zmm_idx_dst, 3);
             swap_scalar(xmm_val(3), xmm_idx(3), xmm_val(4), xmm_idx(4));
             swap_scalar(xmm_val(2), xmm_idx(2), xmm_val(3), xmm_idx(3));
-            movups(xmm_val(0), xmm_val(2));
-            movups(xmm_idx(0), xmm_idx(2));
+            uni_vmovups(xmm_val(0), xmm_val(2));
+            uni_vmovups(xmm_idx(0), xmm_idx(2));
             horize_top1();
         }
     }
 
     // dst: xmm_val(0) and xmm_idx(0)
     // aux: xmm_val(3) and xmm_idx(3)
     inline void horize_top1() {
-        movshdup(xmm_val(3), xmm_val(0));                            // dst:1,2,3,4; aux:2,2,4,4
-        movshdup(xmm_idx(3), xmm_idx(0));
+        uni_movshdup(xmm_val(3), xmm_val(0));                        // dst:1,2,3,4; aux:2,2,4,4
```

Comment:
could you please use similar signature: uni_movshdup() ---> uni_vmovshdup(); update in jit_generator.h

Reply:
Done. Thanks Chenhu!